tests/greenheart/test_openmdao.py

```diff
@@ -21,6 +26,37 @@
 greenheart_config_onshore_filename = Path(__file__).absolute().parent / "input_files" / "plant" / "greenheart_config_onshore.yaml"
 turbine_config_filename = Path(__file__).absolute().parent / "input_files" / "turbines" / "osw_18MW.yaml"
 floris_input_filename_steel_ammonia = Path(__file__).absolute().parent / "input_files" / "floris" / "floris_input_osw_18MW.yaml"
+
+
+from ORBIT.core.library import initialize_library
+
+dirname = os.path.dirname(__file__)
+orbit_library_path = os.path.join(dirname, "input_files/")
+YamlIncludeConstructor.add_to_loader_class(
```

Comment:
Does the `load_yaml` function from `hopp.utilities` not work for these files? Not positive that we need to add a constructor for them.

Reply:
I guess we don't need this. I've removed those lines.